IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml.cs

```diff
@@ -28,40 +29,27 @@ public void OnGet()
             {
                 using (var webClient = new WebClient())
                 {
-                    // grab our JSON text.
-                    string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries");
-
-                    // convert raw text to objects.
-                    List<BreweryCollection> breweryCollections = BreweryCollection.FromJson(jsonString);
-
-                    //Getting data of all Brewery
+                    string BreweriesData = string.Empty;

```

Comment:
Added a Try -catch block to handle exceptions during API calls.

Reply:
Logging is good, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That could be worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().